roles/nighthawk/tasks/edge-tasks.yml

```diff
@@ -0,0 +1,24 @@
+---
+# tasks for edge termination type.
+
+- name: Create service for edge type server pods
+  k8s:
+    definition: "{{ lookup('template', 'http-service.yml.j2') | from_yaml }}"
+  with_sequence: start=0 count={{ workload_args.number_of_routes | int }}
```

Comment:
Multiple instances of workload_args (such as number_of_routes) being referenced without a default value.

Reply:
Added default values to all the workload arguments